src/app/personside/infotabs/ytelser/felles-styling/DescriptionList.tsx

```diff
@@ -15,6 +15,9 @@ export interface DescriptionListEntries {
 }
 
 function getDescriptionlistEntry(term: string, description: DescriptionlistEntry) {
+    if (typeof description === 'boolean') {
+        console.log('Vet ikke hvordan boolsk verdi skal vises for: ', term);
```

Comment:
Hvordan kan dette skje? Er den ikke definert som ikke boolean?

Reply:
Enig med du, men det skjedde likevel run-time. Usikker på hvorfor det ikke ble fanget opp av typescript

Reply:
Weird... verdt å finne ut av?

Reply:
Ser ut som at det er fordi true er en gyldig verdi for ReactNode som aksepteres som parameter for DescriptionListEntries. Kan se om jeg kan bruke noe annet..